src/celldega/qc/__init__.py

```diff
@@ -17,126 +19,76 @@ def qc_segmentation(transcript_metadata_file, transcript_data_file, cell_polygon
-    transcript_metadata_file : str
-        Path to the transcript metadata file, containing details about transcript assignments.
-
-    transcript_data_file : str
-        Path to the transcript data file, containing raw transcript information.
-
-    cell_polygon_metadata_file : str
-        Path to the cell polygon metadata file, containing cell attributes such as area and centroid.
-
-    cell_polygon_data_file : str
-        Path to the cell polygon data file, containing cell polygon geometries.
-
-    dataset_name : str
-        Name of the dataset being processed.
-
-    segmentation_approach : str
-        Description of the segmentation method used.
-
-    subset_interval_y_x : tuple
-        Tuple defining the spatial subset interval for analysis (y, x).
-
-    transform_file : str
-        Path to the transformation matrix file, required for coordinate transformations (used for MERSCOPE/Xenium data).
-
-    cell_boundaries_file : str
-        Path to the cell boundaries file, required for generating cell polygons (used for MERSCOPE/Xenium data).
-
-    path_output_cell_metrics : str
-        Path to save the output CSV file containing cell-level metrics.
-
-    path_output_gene_metrics : str
-        Path to save the output CSV file containing gene-specific metrics.
+    base_path : str
+        Path to the data directory
 
     Returns
     -------
     None
         Outputs two CSV files containing cell-level and gene-specific QC metrics.
 
-    Notes
-    -----
-    - For data from STP, transcript metadata and cell polygons are read directly from provided files.
-    - For Xenium data, transcripts are transformed and cell polygons are derived using provided transformation files.
-    - Metrics calculated include:
-        * Proportion of assigned transcripts
-        * Number of cells
-        * Mean and median transcripts per cell
-        * Mean and median genes per cell
-        * Proportion of empty cells
-    - Gene-specific metrics include proportion of cells expressing a gene, average expression, and assigned transcripts.
-
     Example
     -------
-    qc_segmentation(
-         transcript_metadata_file="path/to/trx_metadata.parquet",
-         transcript_data_file="path/to/trx_data.csv",
-         cell_polygon_metadata_file="path/to/cell_meta.parquet",
-         cell_polygon_data_file="path/to/cell_data.parquet",
-         dataset_name="Dataset1",
-         segmentation_approach="MethodA",
-         subset_interval_y_x=(0, 1000, 0, 1000),
-         transform_file="path/to/transform.mat",
-         cell_boundaries_file="path/to/cell_boundaries.csv",
-         from_stp=True,
-         path_output_cell_metrics="output/cell_metrics.csv",
-         path_output_gene_metrics="output/gene_metrics.csv"
-         )
+    qc_segmentation(base_path="path/to/data")
 
     """
 
     metrics = {}
-    
-    if technology_name == 'STP':
+
+    with open(os.path.join(base_path, "segmentation_parameters.json"), 'r') as parameter_file:
```

Comment:
This seems like a critical file, I suggest adding a line to check the existence of this file to avoid error. Also, it does not seems right to store `segmentation_parameters.json` in current working directory which is `notebooks`. Please consider moving it to a data folder or somewhere that makes more sense. You can always gitignore a folder (in this case, `data/`) that you don't want to push to Github.